packages/react-scripts/template/README.md

```diff
@@ -1066,6 +1066,17 @@ To override this, specify the `homepage` in your `package.json`, for example:
 
 This will let Create React App correctly infer the root path to use in the generated HTML file.
 
+#### Setting Homepage to "."
```

Comment:
Let's change this to "Serving the Same Build from Different Paths" or something similar.